pkg/setup-isoreboot/qemu.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"context"
+
+	"github.com/cybozu-go/log"
+)
+
+func setupQEMU(ctx context.Context, url string) error {
+	// nothing to do
+
+	log.Info("setupQEMU called", map[string]interface{}{
+		"url": url,
```

Comment:
The field name `"url"` is reserved as `log.FnURL`.  Using this field for other generic purpose may confuse log analyzers.  This field may even overwritten by the library.
What about `"iso_url"`?